pkg/mycarehub/infrastructure/database/postgres/gorm/config_test.go

```diff
@@ -124,6 +132,16 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	var genderList pq.StringArray
```

Comment:
remove unused variable. these genderList and clientTypes are not being used in the testsvfixtures